java/src/jmri/util/prefs/JmriConfiguration.java

```diff
@@ -23,7 +23,7 @@
  *
  * @author Randall Wood
  */
```

Comment:
It might be good to also add a note on what this class is doing or what it is used for.